regcore/settings/base.py

```diff
@@ -54,3 +55,6 @@
     from local_settings import *
 except ImportError:
     pass
+
+import django
+django.setup()
```

Comment:
I've removed those lines and they are not necessary after all. The settings overrides for tests took care of the issue this was fixing - I was trying to get the original regcore tests running in their repo after the fixes required for cfgov satellite apps.

I'll get with you and find out which test errors you're getting. Thanks for taking a look at this!